src/foam/nanos/se/SystemEvent.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.se',
+  name: 'SystemEvent',
+
+  implements: [
+    'foam.nanos.auth.CreatedAware'
```

Comment:
also
CreatedByAware
EnabledAware or LifecycleStateAware (lifecycle state is little overkill)